fetch_firmware.sh.m4

```diff
@@ -63,10 +66,9 @@ download_fw() {
 
   # Download piksi_fpga, try the prod variant first, then sdk variant
   fetch $NAP_S3_PATH_PROD/piksi_prod_fpga.bit $FIRMWARE_DIR/piksi_fpga.bit \
-ifelse(M4_BUCKET, swiftnav-releases,
-`    || fetch $NAP_S3_PATH/piksi_sdk_fpga.bit $FIRMWARE_DIR/piksi_fpga.bit \
-    || error "failed to download piksi_fpga.bit"',
-`    || error "failed to download piksi_fpga.bit"')
+ifelse(M4_BUCKET, swiftnav-releases, `    || fetch_sdk_fpga \
```

Comment:
not really sure what is going on here, syntax is foreign to me but I'm guessing there was a reason to refactor the format

Reply:
Removed some duplication and made a helper that generates the `.sha1sum` file that we use to detect if we're doing an SDK build.